### DISCUSSION TITLE
[Config Support]: Restream camera + detections

### DISCUSSION THREAD

**Discussion by hassanbadawy** [blakeblackshear/frigate#13479] [Config Support] [answered]
### Describe the problem you are having

Dear All,
I've two questions:
1- How can I restream camera including detections to other softwares on rtsp?
2- How can I live view camera+detections on main dashboard (not debug camera?

### Version

latest

### Frigate config file

```yaml
main config
```

### Relevant Frigate log output

```shell
no need
```

### Relevant go2rtc log output

```shell
no need
```

### Operating system

Debian

### Install method

Docker Compose

### Object Detector

TensorRT

**Comment by hawkeye217** [comment, marked as answer]
Restreaming or viewing the debug view on the dashboard is not supported and not possible at this time.

**Reply by hassanbadawy** [reply]
Okay,  many thanks.

**Reply by hassanbadawy** [reply]
It looks I've to stick with Nvidia deepstream for a long time :(
